src/test/staking/block_validator_tests.cpp

```diff
@@ -18,8 +19,46 @@ namespace {
 std::unique_ptr<blockchain::Behavior> b =
     blockchain::Behavior::NewFromParameters(blockchain::Parameters::TestNet());
 
+struct KeyFixture {
+  CExtKey ext_key;
+  CPubKey pub_key;
+  std::vector<unsigned char> pub_key_data;
+};
+
+KeyFixture MakeKeyFixture(const std::string& seed_words = "cook note face vicious suggest company unit smart lobster tongue dune diamond faculty solid thought") {
```

Comment:
It looks like `lint-clang-format.py` is no longer executed on Travis

Reply:
So sad.

Reply:
Maybe it is just not applied to `src/test/staking`.

Reply:
The linter will be run again in https://github.com/dtr-org/unit-e/pull/1101